.travis.yml

```diff
@@ -1,7 +1,9 @@
 sudo: false
 
 language: c
-
+arch:
+  - AMD64
+  - ppc64le
 compiler:
```

Comment:
thank you! just cosmetic change: do not remove empty line separator and also add new line before compiler: section

Reply:
Thank you, updated recommended changes please review it.

Reply:
Ok, fine now. Could you squash fixup change? I will then merge it.

Reply:
Thanks, squashed it to single commit.